docs/source/nodeoperations/index.rst

```diff
@@ -3,20 +3,10 @@ Node operations using Scylla Operator
 ======================================
 
 .. toctree::
-   :hidden:
```

Comment:
let's also lower maxdepth to 1 to align with what it looked like before

Reply:
done